core/src/main/java/org/zenoss/zep/zing/ZingPublisher.java

```diff
@@ -19,19 +19,40 @@
 import com.google.pubsub.v1.PubsubMessage;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.zenoss.zep.zing.ZingConfig;
-import org.zenoss.zep.zing.ZingEvent;
 import org.zenoss.zing.proto.event.Event;
 
-public abstract class ZingPublisher {
+public class ZingPublisher {
```

Comment:
Are you sure you don't want this to be abstract? will it work if instantiated at this level (i.e. not via a subclass)? If not, keep it as an abstract class